en/08_Changelogs/6.2.0.md

```diff
@@ -8,6 +8,8 @@ title: 6.2.0 (unreleased)
 
 - [Features and enhancements](#features-and-enhancements)
   - [Accessibility improvements](#accessibility-improvements)
+  - [Unsaved changes indicator](#unsaved-changes-indicator)
```

Comment:
This was missing though the section was added. Figured I may as well just add it in rather than spawning a whole separate issue just for this